### PR TITLE
chore: Create issuer-wasm docker image

### LINKED ISSUE #4: Create issuer-wasm docker image

### PR DESCRIPTION
**Pull request by fqutishat**
closes #4

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,15 @@ WASM_COMPONENTS        = cmd/components
...
 GOBIN_PATH             = $(abspath .)/build/bin
 
+# GO version
+ALPINE_VER ?= 3.10
+GO_TAGS    ?=
+GO_VER     ?= 1.13.1
+
+# Namespace for the images
+DOCKER_OUTPUT_NS         ?= docker.pkg.github.com
+ISSUER_WASM_IMAGE_NAME   ?= trustbloc/edge-agent/issure-agent-wasm
```
Comment (troyronda) [inline Makefile]: typo

File: test/bdd/fixtures/agent-wasm/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+version: '2'
+
+services:
+
+  issuer.agent.example.com:
+    container_name: issure.agent.example.com
```
Comment (troyronda) [inline test/bdd/fixtures/agent-wasm/docker-compose.yml]: typo

Comment (troyronda) [reply test/bdd/fixtures/agent-wasm/docker-compose.yml]: There are a more similar typos (issure)

File: Makefile (hunk 1 above)
Comment (fqutishat) [reply Makefile]: I fixed all typo

File: test/bdd/fixtures/agent-wasm/docker-compose.yml (hunk 2 above)
Comment (fqutishat) [reply test/bdd/fixtures/agent-wasm/docker-compose.yml]: I fixed all typo

**Pull request merged**